nifi-commons/nifi-property-utils/src/main/java/org/apache/nifi/properties/BootstrapProperties.java

```diff
@@ -33,13 +34,22 @@
     public enum BootstrapPropertyKey {
         SENSITIVE_KEY("bootstrap.sensitive.key"),
         HASHICORP_VAULT_SENSITIVE_PROPERTY_PROVIDER_CONF("bootstrap.protection.hashicorp.vault.conf"),
-        AWS_KMS_SENSITIVE_PROPERTY_PROVIDER_CONF("bootstrap.protection.aws.kms.conf");
+        AWS_KMS_SENSITIVE_PROPERTY_PROVIDER_CONF("bootstrap.protection.aws.kms.conf"),
+        CONTEXT_LOCATION_MAPPING("bootstrap.protection.context.mapping.");
```

Comment:
Adjusting the name to indicate that it is a prefix would help clarify the reason for the trailing period:
```suggestion
        CONTEXT_MAPPING_PREFIX("bootstrap.protection.context.mapping.");
```